README.md

```diff
@@ -14,7 +14,6 @@ simply run:
 
 ```sh
 npm install
-grunt
```

Comment:
Since `npm run build` was added to package.json as the `prepublish` script, it will run as part of the `npm install` lifecycle. Granted, this behavior is deprecated and will be removed in `npm` v5 (see [here](https://github.com/npm/npm/blob/30d75e738b9cb7a6a3f9b50e971adcbe63458ed3/doc/misc/npm-scripts.md#prepublish-and-prepare) for details), so this part of the documentation will need to be changed anyway.

What if I changed the line to this?

```sh
npm run build # if you're using npm v5+
```